### ISSUE TITLE
U-Grade - grading options for different retail partner levels

### ISSUE THREAD

**Issue by rodolfomartinez** [LuchaComics/cps-frontend#160]
**Summary:**
When a U-Grade submission is made, and either the CPS Number cale os CPS percentage scale is chosen, the following rules should apply:

Level 1 Retail partners only get up to 9.6 for CPS Number Scale, or 96% for CPS percentage scale
Level 2 and 3 Retailer partners get up to 9.8 for CPS Number Scale or 98% for CPS Percentage Scale

**Comment by rodolfomartinez** [comment]
@bartmika I do not think that levels are saving correctly. I upgraded a retailer to Level 2 and they still could not select a 9.8 grade. When I went back to their profile to try and upgrade to 3, the level dropdown was set back to select one (as if they had no level) and several fields reported missing values - this seems to only happen when a level is changed.
![image](https://github.com/LuchaComics/cps-frontend/assets/7537946/92993c3d-9459-4de5-a697-78c5b377dbcf)
